app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -3,7 +3,9 @@
 
 /* eslint-disable max-lines */
 
+import {intlShape} from 'react-intl';
 import keyMirror from '@mm-redux/utils/key_mirror';
+import {createCallRequest} from '@utils/apps';
```

Comment:
Any imports added to this file should go in `./app_command_parser_dependencies` so the webapp can use the same dependencies. This excludes the external dependency `react-intl` here.

Reply:
I accidentally have the `@mm-redux` here which should also go in the dependencies file